developer-docs-site/docs/concepts/coin-and-token/aptos-token-overview.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Aptos Token Standard Overview"
+id: "aptos-token-overview"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token Standard Overview
+
+:::tip Aptos token standard
+Also see the official token standard [Aptos Token](/concepts/coin-and-token/aptos-token.md).
+:::
+
+## Introduction
+One of the major application of blockchain is token, which requires a standard defining
+the pattern a token must comply with. The Aptos team and communities also have been moving
+fast and steadily to support various projects and creators to launch their tokens on aptos
+blockchain. As a new L1 blockchain equipped with Move, a relatively new language exclusively
+designed for the safety of smart contract, our token standard inevitably differs from other
+blockchains more or less. In this doc, we will briefly cover the comparison between aptos
+ethereum and solana token standard.
+
+### Account Model
```

Comment:
I think adding a diagram for the different models below would make this even easier to follow. Especially the comparison on multiple accounts used for different purposes.

Reply:
tbh, I prefer to not to include the account models because it could be a standalone doc/blog all by itself. Either ethereum or solana has rich docs describe their account model and even with a diagram I don't think n00bs can get the idea easily. For experienced developers who are familiar with both, it is just a refreshment of their rusty memory.